Gemfile

```diff
@@ -23,7 +23,7 @@ end
 group(:development, :test) do
   gem 'json'
   gem 'lock_manager', *lock_manager_location_for(ENV['LOCK_MANAGER_LOCATION'] || '>= 0')
-  gem 'packaging', github: 'puppetlabs/packaging', branch: 'master'
+  gem 'packaging', github: 'puppetlabs/packaging', branch: '1.0.x'
```

Comment:
Not really. Yes, there's puppet5 specific stuff, but a lot of it is backward compatible. See the comment I made on this PR earlier today.